src/realm.js

```diff
@@ -205,6 +207,7 @@ export class Realm {
   modifiedBindings: void | Bindings;
   modifiedProperties: void | PropertyBindings;
   createdObjects: void | CreatedObjects;
+  impureObjects: void | ImpureObjects;
```

Comment:
So what is the meaning of `impureObjects`? Reading through the code... newly created objects within the most enclosing `evaluatePure` scope which were exposed to abstract function calls?

Reply:
Newly created objects within evaluatePure. They may not have been exposed to abstract function calls yet. These are the objects that have the potential to become leaked.